proof/refine/CSpace_R.thy

```diff
@@ -56,6 +60,81 @@ lemmas untyped_inc' = untyped_inc'
 
 end
 
+(* FIXME arch-split: move requalification and same_master_descendants to CSpace_R
+   when arch-splitting it, these can be generic *)
```

Comment:
Good job thinking to move these but I think this FIXME can be removed now.

Reply:
yup, cheers